lib/improver/nowcasting/pysteps_advection.py

```diff
@@ -34,13 +34,18 @@
 from datetime import timedelta
 
 from iris.coords import AuxCoord
-from pysteps.extrapolation.semilagrangian import extrapolate
 
+from improver.metadata.amend import (amend_attributes, set_history_attribute)
 from improver.utilities.spatial import (
     check_if_grid_is_equal_area, calculate_grid_spacing)
 from improver.utilities.temporal import (
     iris_time_to_datetime, datetime_to_iris_time)
 from improver.nowcasting.utilities import ApplyOrographicEnhancement
+from improver.utilities.stdout_trap import Capture_StdOut
+
+# PySteps is so proud of itself - trap the routine StdOut message on import
```

Comment:
Printing when importing!?!

Reply:
Actually I've just had a look - could someone point out where this print is coming from? (I can't see it at a glance)

Reply:
Ah I missed this.  PySteps is of course rightly proud - but we should probably rephrase this comment!!!

Reply:
And yeah.  It says "Look I found a pystepsrc file at </user/specific/path/>!"  Or words to that effect.  Which is probably handy for command-line interactivity, but not so much for real-time running.

Reply:
I've changed the comment.